SwiftLoggerSampleApp/SwiftLoggerSampleApp.xcodeproj/project.pbxproj

```diff
@@ -319,7 +319,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"SwiftLoggerSampleApp/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 9FD3J9E394;
```

Comment:
Tady nemuzes prece davat do obecnyho repositare svuj tym.

Reply:
neslo mi to kvuli tomu buildit, vratil jsem to zpatky